master/deployment.md

```diff
@@ -19,4 +19,32 @@ $ php artisan route:cache
 $ php artisan migrate --force
 $ php artisan horizon:purge
 $ php artisan storage:link
+$ php artisan horizon:install
+$ php artisan horizon:assets
 ```
+
+### Background Task Runner
+
+Create the file `/etc/systemd/system/pixelfed-queue.service` with the following content:
+
+```
+[Unit]
+Description=Pixelfed task queueing via Laravel Horizon
+After=network.target
+Requires=php7.2-fpm.service
+Requires=redis.service
+Requires=postgresql.service
+Wants=nginx.service
+
+[Service]
+Type=simple
+ExecStart=/usr/bin/php /home/pixelfed/artisan horizon
```

Comment:
indicate artisan needs to be wherever pixelfed was installed

Reply:
There's a paragraph describing this.